### PR TITLE
UIREC-7 Create Receive Button

### PR DESCRIPTION
**Pull request by AndreiShumski2**
screenshot_1

https://issues.folio.org/browse/UIREC-7

### CODE REVIEW

File: src/components/PurchaseOrder/PO.js [hunk 1]
```diff
@@ -242,10 +241,21 @@ class PO extends Component {
     const orderNumber = get(order, 'po_number', '');
     const poLines = get(order, 'po_lines', []);
     const workflowStatus = get(order, 'workflow_status');
-    const isCloseOrderButtonVisible = orderId && workflowStatus === WORKFLOW_STATUS.open;
+    const isWorkflowStatusOpen = workflowStatus === WORKFLOW_STATUS.open;
+    const isCloseOrderButtonVisible = orderId && isWorkflowStatusOpen;
+    const isReceiveButtonVisible = orderId && isWorkflowStatusOpen && (poLines.length > 0);
```
Comment (aliaksei-chumakou) [inline src/components/PurchaseOrder/PO.js]: I think `poLines.length` is not enough. On the meeting Dennis mentioned physical quantity and electronic. So it looks like you supposed to look (maybe reduce) `poLines` to have these values more than 0

File: src/components/PurchaseOrder/PO.js [hunk 2]
```diff
@@ -242,10 +241,21 @@ class PO extends Component {
...
     const lastMenu = (
       <PaneMenu>
         <IfPermission perm="purchase_order.item.put">
+          {isReceiveButtonVisible && (
+            <Button
+              buttonStyle="primary"
+              marginBottom0
+              style={{ marginRight: '10px' }}
```
Comment (aliaksei-chumakou) [inline src/components/PurchaseOrder/PO.js]: these `marginRight` don't work well with RTL feature. Maybe it makes sense to use flebox spacing here between buttons

File: src/components/PurchaseOrder/PO.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/components/PurchaseOrder/PO.js]: Ok. Now physical and/or electronic quantity are required fields to fill in any case.

Comment (aliaksei-chumakou) [reply src/components/PurchaseOrder/PO.js]: The key word here `Now`. Who knows what order types could be in a future, and Dennis was clear regarding to look at quantity fields

Comment (AndreiShumski2) [reply src/components/PurchaseOrder/PO.js]: I got it, thank you.

**Pull request merged**